pkg/util/apiserver/apiserver.go

```diff
@@ -180,6 +182,16 @@ func (as *APIServer) addSerializedHandler(pattern string, m k8sruntime.Object) {
 	}))
 }
 
+// Copied from https://github.com/kubernetes/kubernetes/pull/101123 until the referenced PR is merged
```

Comment:
nit: Can you put this at the bottom of the comment block so that the comment starts with the method name (to make generated documentation look correct)?

Reply:
Done, thanks. Wonder why my linter didn't catch that, it usually does for method docs. Maybe because it's a private method.

Reply:
Thanks!